### PR TITLE
CJM-16883: XDM changes for STO & Test profile

### PR DESCRIPTION
**Pull request by shubhirastog**
https://jira.corp.adobe.com/browse/CJM-16883

### CODE REVIEW

File: extensions/adobe/experience/customerJourneyManagement/messageprofile.schema.json [hunk 1]
```diff
@@ -37,6 +37,16 @@
...
         },
+        "https://ns.adobe.com/experience/customerJourneyManagement/messageProfile/isExecutionSendTimeOptimized": {
+          "title": "isExecutionSendTimeOptimized",
+          "type": "boolean",
+          "description": "True if the message execution is send time optimized."
+        },
+        "https://ns.adobe.com/experience/customerJourneyManagement/messageProfile/isTestProfile": {
+          "title": "isTestProfile",
+          "type": "boolean",
+          "description": "True if the message is sent as Test Profile."
```
Comment (sawhneyh) [inline extensions/adobe/experience/customerJourneyManagement/messageprofile.schema.json:48]: Should we rename `if the message is sent as Test Profile` to `if the message is sent to a Test Profile`

**Comment by shubhirastog** [comment]
@prabhum2 Please review. Also, I see people making changes to 4 files which include files from docs folder. Is this should be the practice to follow or the docs are autogenerated?

**Review by prabhum2 — APPROVED** [review]

**Comment by prabhum2** [comment]
> @prabhum2 Please review. Also, I see people making changes to 4 files which include files from docs folder eg. in PR - https://github.com/adobe/xdm/pull/1217/files. Is this should be the practice to follow or the docs are autogenerated?

Docs are autogenerated . You need to make changes to Schema json and example json files if necessary

**Review by sawhneyh — APPROVED** [review]

**Comment by shubhirastog** [comment]
@cmathis & @prabhum2 Could we please merge this PR?

**Review by jainaks — APPROVED** [review]

**Pull request merged**